documents/store-operations/bopis/order-details-page/README.md

```diff
@@ -11,26 +11,47 @@ Users can access the Order Details page by clicking on any order card from any o
 
 On clicking on any order card in the `Open`, `Packed` or `Completed` tab, users will be directed to the order details page. The order details page displays key details of an order and also enables store associates to perform the certain functions.
 
-### Reject Orders
+### Item Details
+
+At the top of the Order Details page, this section displays the order name and order ID. It lists all items in order, along with their product image, primary, and secondary identifiers. If an item is part of a kit, a "KIT" tag is shown next to the item name. Each item has two icons: a box icon that shows the quantity on hand and a bin icon to reject the item. 
 
-During peak hours or promotional events, brick-and-mortar retail locations often face increased in-store traffic, heightening the risk that the requested item for pick-up may become unavailable due to in-store purchases. In such instances, store associates may need to reject an order or reject an order partially in case the order item inventory is insufficient at the store. Partial order rejection feature can be enabled using the toggle on the settings page.
+The action buttons shown depend on the order status. When the order is in Open status, buttons like Ready for Pickup and Reject Items are available. After the order is marked Ready for Pickup, these buttons change to Cancel Item and Handover. If the order is marked as Ready for Pickup, the only action available is to cancel it. If an item is damaged or out of stock, it should be rejected before packing. After packing, the order is marked Ready for Pickup, and an email is sent to the customer. So, unpacking or rejecting is not allowed at this stage.
```

Comment:
Don't use words "Like" if you know the exact button being displayed.
After ready for pickup, the only option is to cancel an order?

Overall this is way too wordy. Simplify it and make it easier for people to find what they're trying to read.